Source/UI/Controllers/CardViewController.swift

```diff
@@ -108,10 +109,12 @@ public class CardViewController: UIViewController,
         registerKeyboardHandlers(notificationCenter: notificationCenter,
                                  keyboardWillShow: #selector(keyboardWillShow),
                                  keyboardWillHide: #selector(keyboardWillHide))
-
+        
         if suppressNextLog {
             suppressNextLog = false
         } else {
+            checkoutApiClient?.logger.add(metadata: (checkoutApiClient?.correlationID())!,
```

Comment:
@daven-gomes-cko @harry-brown-cko I wanted to get rid of the forced unwrap here. But for some reason the checkoutApiClient is option on the view controller itself. Any idea why checkoutApiClient optional when creating the CardViewController.

Reply:
Because the initialisers aren't setup to guarantee that it's there 😔

Reply:
For now I have unwrapped this optional and doing a return of nil as other places in this class which is safe. Ideally we should be return a error back.